components/builder-depot/src/backend/s3.rs

```diff
@@ -210,6 +212,7 @@ impl S3Handler {
                 Counter::UploadFailures.increment();
                 debug!("{:?}", e);
                 let end_time = PreciseTime::now();
+                info!("Upload failed for: {}", &path_attr);
```

Comment:
Make this a warn! and add in the error code (and then can remove the debug line above)